backend/src/tasks/ecs-client.ts

```diff
@@ -102,7 +102,8 @@ class ECSClient {
             `WORKER_SIGNATURE_PRIVATE_KEY=${process.env.WORKER_SIGNATURE_PRIVATE_KEY}`,
             `ELASTICSEARCH_ENDPOINT=${process.env.ELASTICSEARCH_ENDPOINT}`,
             `AWS_ACCESS_KEY_ID=${process.env.AWS_ACCESS_KEY_ID}`,
-            `AWS_SECRET_ACCESS_KEY=${process.env.AWS_SECRET_ACCESS_KEY}`
+            `AWS_SECRET_ACCESS_KEY=${process.env.AWS_SECRET_ACCESS_KEY}`,
+            `LG_API_KEY=${process.env.LG_API_KEY}`
```

Comment:
Can you also add all the necessary scaffolding within the infrastructure folder (in the `.tf` and `.tfvars` files) so that LG_API_KEY is properly set? You could just do a ctrl+f for HIBP_API_KEY in the diff for https://github.com/cisagov/crossfeed/pull/1058/files and ensure that similar changes are added to this PR as well.